src/game.js

```diff
@@ -1,74 +1,81 @@
-var Game = {
-    players: [],
-    currentPlayer: 0,
-    board: {},
-    dice: [],
-    chanceCards: [],
-    communityChestCards: [],
-    doublesCount: 0,
+var Utils = require("./utils.js");
 
-    randomizeCurrentPlayer: function() {
-        this.currentPlayer = Math.floor(Math.random() * this.players.length);
-    },
+function Game(players, chance, communityChest) {
+    this.players = players,
+    this.currentPlayer = 0;
+    this.board = {};
+    this.dice = [];
+    this.chanceCards = chance;
+    this.communityChestCards = communityChest;
+    this.doublesCount = 0;
+    this.randomizeCurrentPlayer();
 
-    rollDice: function(numberOfDice, sides) {
-        if (!sides) {
-            sides = 6;
-        }
-        this.dice = [];
-        for (i = 0; i < numberOfDice; i++) {
-            roll = Math.floor(Math.random() * sides) + 1;
-            this.dice.push(roll);
-        }
-    },
+    Utils.shuffle(this.chanceCards);
+    Utils.shuffle(this.communityChestCards);
+};
+
+Game.prototype.randomizeCurrentPlayer = function() {
+    this.currentPlayer = Math.floor(Math.random() * this.players.length);
+};
 
-    movePlayer: function() {
-        this.rollDice(2);
-        if (this.players[this.currentPlayer].inJail) {
-            this.players[this.currentPlayer].turnsInJail += 1;
-            if (this.isDoubles(this.dice)) {
-                this.players[this.currentPlayer].releaseFromJail();
-                this.move();
-            } else if (this.players[this.currentPlayer].turnsInJail === 3) {
-                this.players[this.currentPlayer].payBail();
-                this.move();
-            }
-        } else {
+Game.prototype.rollDice = function(numberOfDice, sides) {
+    if (!sides) {
+        sides = 6;
+    }
+    this.dice = [];
+    for (i = 0; i < numberOfDice; i++) {
+        roll = Math.floor(Math.random() * sides) + 1;
+        this.dice.push(roll);
+    }
+};
+
+Game.prototype.movePlayer = function() {
+    this.rollDice(2);
+    if (this.players[this.currentPlayer].inJail) {
+        this.players[this.currentPlayer].turnsInJail += 1;
+        if (this.isDoubles(this.dice)) {
+            this.players[this.currentPlayer].releaseFromJail();
+            this.move();
+        } else if (this.players[this.currentPlayer].turnsInJail === 3) {
+            this.players[this.currentPlayer].payBail();
             this.move();
```

Comment:
could this.move() just go outside this entire if... if..else? looks like you call it in every case